README.md

```diff
@@ -36,6 +38,41 @@ Initially, there are created 2 default roles and users in the database:
 
 To change initial data edit SeedValues.json file
 
+## Use
+
+To receive access and refresh token by user credentials send the following request:
+POST /connect/token
+```csharp
+var values = new Dictionary<string, string>
+{
+	{ "grant_type", "password" },
+	{ "password", "userPassword" },
+	{ "username", "userName" },
+	{ "scope", "offline_access" },
+	{ "client_id", "clientId" },
+	{ "client_secret", "clientSecret" }
+};
+
+var content = new FormUrlEncodedContent(values);
+var response = await httpClient.PostAsync($"{authServerUrl}/connect/token", content);
+```
+
+To receive access and refresh token by refresh token send the following request:
```

Comment:
Let's change the beginning of the string on 'To receive refresh token by user credentials'